dashboard/app/models/unit_group.rb

```diff
@@ -500,6 +500,22 @@ def self.latest_stable_version(family_name, locale: 'en-us')
     end
   end
 
+  def supported_locale_codes
+    locales = default_unit_group_units.first&.script&.supported_locales || []
```

Comment:
To make sure I'm following- does the supported_locales list in the first unit represent the translations of the rest of its units here?

Reply:
Ahh thank you for that. I would need to know more about ruby to understand if this works in-place.